CHANGES.rst

```diff
@@ -381,6 +381,8 @@ astropy.uncertainty
 astropy.units
 ^^^^^^^^^^^^^
 
+- Fix parsing of numerical powers in FITS-compatible units. [#8251]
```

Comment:
changelog only needed to be added to the section a PR is milestoned to.

Reply:
Ah, I misunderstood.  It sounded like @mhvk wanted me to add two entries.